apps/nrm/behaviours/resume-form-session.js

```diff
@@ -78,6 +80,9 @@ module.exports = superclass => class extends superclass {
       }
       req.sessionModel.set('redirect-to-reports', false);
       return super.getValues(req, res, next);
-    });
+    } catch (error) {
+      req.log('info', `External ID: ${externalID}, Error Saving Session: ${error}`);
```

Comment:
We have been using the logger tool from HoF, just curious as to why logger is not use in this try catch block? Logger is also not used in the save-form-session.js file.

Reply:
When we have a req object available, we can use req.log. In hof, req.log uses the hof logger in conjunction with another logging middleware called Morgan which allows us to access some additional information in the logs such as the session id. If a function does not use of the req object, then using the hof logger is enough.